src/views/transactions/components/account-selector/account-selector.view.jsx

```diff
@@ -18,14 +18,19 @@ function AccountSelector({
   onAccountClick,
 }) {
   const classes = useAccountSelectorStyles();
-  const safePendingDeposits = pendingDeposits ? pendingDeposits : [];
-  const disabledTokenIds = safePendingDeposits
+  const disabledTokenIds = pendingDeposits
     .filter((deposit) => deposit.type === TxType.CreateAccountDeposit)
     .map((deposit) => deposit.token.id);
 
   React.useEffect(() => {
     if (accountsTask.status === "pending" && pooledTransactionsTask.status === "successful") {
-      onLoadAccounts(undefined, pooledTransactionsTask.data, fiatExchangeRates, preferredCurrency);
+      onLoadAccounts(
+        undefined,
+        pooledTransactionsTask.data,
+        pendingDeposits,
```

Comment:
`onLoadAccounts` doesn't need `pendingDeposit`. Now it's not expecting that value